CONTRIBUTING.md

```diff
@@ -8,12 +8,17 @@ This page instructs how to contribute code changes to DevTools.
 > If you just want to test newest functionality, follow
 [beta testing guidance](https://github.com/flutter/devtools/blob/master/BETA_TESTING.md).
 
-To contribute code you must complete the
+Before contributing code:
+
+1. Complete the
 [Contributor License Agreement](https://cla.developers.google.com/clas).
-You can do this online, and it only takes a minute. If you've never submitted code before,
-you must add your (or your organization's) name and contact info to the [AUTHORS](AUTHORS)
+You can do this online, and it only takes a minute.
+
+2. Make sure your (or your organization's) name and contact info is added to the [AUTHORS](AUTHORS)
 file.
```

Comment:
I think we can remove this - we don't follow this practice. @jacob314 please correct me if I am wrong.

Reply:
Yeah I don't know of us ever following that practice. Does any repo under the Flutter org do this?

Reply:
Out of top 5 Flutter repos I checked only flutter-intellij has this instruction and follows the rule:

https://github.com/flutter/flutter-intellij/blob/master/CONTRIBUTING.md

https://github.com/flutter/flutter-intellij/blob/master/AUTHORS

Reply:
deleted